templates/summary/_block/header.html.twig

```diff
@@ -0,0 +1,21 @@
+<section class="l__wrapper--medium">
+    {% block summary_left_part %}
+        <div>
+            <div>Ajouter une photo</div>
+            <h2 class="text--large b__nudge--bottom-tiny">{{ summary.member.fullname }}</h2>
+            <div class="text--body text--gray">{{ summary.member.age }} ans, {{ summary.member.cityName }}</div>
+        </div>
+    {% endblock %}
+    {% block summary_right_part %}{% endblock %}
+    {% block summary_bottom_part %}
+        <div>
+            <div>
+                {{ summary.member.activatedAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM Y') }}<br>
+                Date d'adhésion
+            </div>
+            <div>
+                23 <br> participations à des events
```

Comment:
Indeed, but this is the topic of another ticket as there are multiple places in the page where we have to find past data about the user.